source/docs/embark_commands.md

```diff
@@ -49,6 +49,7 @@ Option | Description
 `-b`, `--host` | `host` to run the dev webserver (default: localhost)
 `--noserver` | disable the development webserver
 `--nodashboard` | simple mode, disables the dashboard
+`--nobrowser` | prevent the development webserver from automatically opening a web browser
```

Comment:
I’m not sure about the wording here. The web server doesn’t open the browser, embark does.

Reply:
I definitely see your point. What subconsciously shaped my choice of words: a DApp's `webserver.js` config file is where `openBrowser: true|false` is specified, and embark's `lib/modules/webserver` is on the receiving end of the cli/config options and the `browser open` command, so I perceived "the development webserver" as what's responsible for the "opening" action.

If it seems better to to say "embark" vs. "the development webserver", I'll amend the cli-help output and docs accordingly.